lxd/instance/drivers/driver_qemu.go

```diff
@@ -2627,6 +2653,12 @@ func (d *qemu) Stop(stateful bool) error {
 	monitor, err := qmp.Connect(d.monitorPath(), qemuSerialChardevName, d.getMonitorEventHandler())
 	if err != nil {
 		// If we fail to connect, it's most likely because the VM is already off.
+		// but it could also be because the qemu process is hung, check for that
+		pid, _ := d.pid()
```

Comment:
Sure I think that's fine, as long as all the callers of `pid()` only want the pid of the qemu process if it's currently running, as opposed to trying to determine what the pid of qemu process that was started for the instance, even if it has since exited.

Reply:
@s-alty yes the use of the `pid` field in the status output is for the running pid, so if its not running we don't actually want to see a pid being outputted.